app/views/models/index.html.erb

```diff
@@ -0,0 +1,61 @@
+<div class="row row-cols-2">
+  <div class="col-9">
+    <div class="row row-cols-2 align-items-baseline">
+      <div class="col col-auto me-auto">
+        <h1><%= # @library.name
+        %></h1>
+      </div>
+      <div class="col col-auto ">
+        <a href="?order=name" class="btn <%= session["order"] == "name" ? "btn-primary" : "btn-light" %> btn-sm" title="sort by name"><i class="bi bi-book"></i></a>
+        <a href="?order=recent" class="btn <%= session["order"] == "recent" ? "btn-primary" : "btn-light" %> btn-sm" title="sort by time"><i class="bi bi-clock"></i></a>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="row row-cols-2">
+  <div class="col-9">
+    <div class="row row-cols-1 row-cols-md-2 row-cols-lg-3 mb-4">
+      <%= render partial: 'model', collection: @models %>
+    </div>
+    <% if current_user.pagination_settings["models"] %>
+      <%= paginate @models %>
+    <% end %>
+  </div>
+  <div class="col-3">
+
+    <%= card :secondary, "Actions" do %>
+      <%= link_to 'Delete Library', params[:library], method: :delete, data: {confirm: "Are you sure? Your files will not be affected."}, class: "btn btn-danger mb-3 me-3" if params[:library] %>
+      <%= link_to "Bulk Edit", edit_models_path(params_passthrough()), class: 'btn btn-secondary mb-3 me-3' %>
+    <% end %>
+
+    <% if !params_passthrough().empty?() %>
+      <%= card :secondary, "Limits" do %>
```

Comment:
Perhaps "filters" rather than "limits"?

Reply:
Perfect.  It didn't feel right, but I was coming up blank (even though I used that word in other places).

Reply:
```suggestion
      <%= card :secondary, "Filters" do %>
```